docusaurus-docs/conda-store-ui/tutorials/create-envs.md

```diff
@@ -83,6 +83,22 @@ To install packages published only on [PyPI][pypi] using [`pip`][pip], include a
 
 -->
 
+### Set environment variables
+
+:::note
+This feature is available *after* conda-store-ui version 2024.1.1.
+
+Currently, only the `CONDA_OVERRIDE_CUDA` environment variable can be specified, which allows setting the CUDA version for building packages with GPU support.
+Learn more in the [conda documentation][conda-docs-override-packages]
+:::
+
+You can set environment variables in the YAML-editor with the following syntax:
```

Comment:
```suggestion
You can set environment variables in the YAML editor with the following syntax:
```

Reply:
This is just to have consistent spelling, it's "YAML editor" in other parts of the doc.